HISTORY.rst

```diff
@@ -2,12 +2,14 @@
 
 History
 -------
+
 v0.32 (unreleased)
 ..................
 * add model name to ``ValidationError`` error message, #676 by @dmontagu
 * **breaking change**: remove ``__getattr__`` and rename ``__values__`` to ``__dict__`` on ``BaseModel``,
   deprecation warning on use ``__values__`` attr, attributes access speed increased up to 14 times, #712 by @MrMrRobat
 * support ``ForwardRef`` (without self-referencing annotations) in Python3.6, #706 by @koxudaxi
+* implement ``schema_extra`` in ``Config`` sub-class, #663 by @tiangolo
```

Comment:
Oh! Sorry, didn't notice I put it in the wrong place.

Reply:
no problem, we need #665.